articles/sessions/concepts/cookie-attributes.md

```diff
@@ -0,0 +1,124 @@
+---
+description: Understand how browser changes, such as the samesite cookie attribute, affects your web applications that embed content from third-party domains. 
+topics:
+  - saml
+  - identity-providers
+  - oidc
+  - cookies
+  - samesite
+  - google-chrome
+  - microsoft-edge
+contentType:
+  - concept
+useCase:
+  - third-party-content
+  - use-samesite-cookie-attribute
+  - maintain-browser-compatiblity
+---
+# sameSite Cookie Attributes
+
+Cookies, primarily used for authentication and maintaining sessions, can be secured by setting cookie attributes. [Auth0 cookies](https://auth0.com/privacy#cookie-policy) are secured by setting certain attributes. Auth0 uses cookies across origins for many reasons: 
+
+* OIDC Enterprise with `form_post`
+* SAML HTTP-POST Binding
+* Web message (aka `checkSession`)
+
+`sameSite` is a cookie attribute that acts as a restriction for the browser’s cookie jar telling it to omit sending the associated cookie key+value pair based on the type of interaction that triggered the HTTP request.
+
+Other cookie attributes you are already familiar include:
+
+* `httpOnly` - makes a cookie only be sent with HTTP Requests and not readable using javascript’s document.cookie.
+
+* `secure` - makes the browser only send the cookie to a secure context, “secure” is defined per browser vendor but in reality it means HTTP over TLS (https).
+
+* `max-age / expires` - controls whether the cookie is “session” cookie /dropped when your browser terminates the browsing session/, or one that is “persistent” /persists browsing session termination/.
+
+Aside from cookies set by the browser’s javascript context, these options are provided by the server with its HTTP responses as part of the set-cookie headers, the browser, upon receipt of the responses parses these and maintains its cookie jar according to them.
+
+## `sameSite` attributes changes
+
+Effective around February 2020, Google Chrome (v80) will change its cookie handling. Two things are changing with this planned release:
+
+* Cookies set without `sameSite` attribute will be defaulted to `lax` instead of the current implicit value `none`.
+
+* Cookies set with `sameSite=none` must also be secure or they’ll be rejected to be saved in the browser’s cookie jar. 
+
+The reason for this change is to improve default security and prevent CSRF attacks. 
+
+### Changes Auth0 has made
+
+Auth0 has made changes to the following cookies: 
+
+* `auth0`: User session
+* `auth0-mf`: Multi-Factor information
+* `did`: Device/User agent identifier
+
+For all the cookies: 
+* We will set the `sameSite=None` attribute and the cookie will always require to use HTTPS regardless of the environment.
+* We will set a fallback cookie `*_compat` for legacy browsers.
```

Comment:
Since this is a public document I would be more specific
```suggestion
* We will set fallback cookies for each cookie to be used by legacy browsers that do not support `sameSite=none`. The name of the new cookies are `auth0_compat`, `auth0-mf_compat` and `did_compat`.
```